### PR TITLE
Small changes for the pres tomorrow

### PR DESCRIPTION
**Pull request by keponk**
Mainly:
* typo spring/fall semester
* fixed link for homework
* suggested firefox instead of chrome :see_no_evil:  (hope that's ok, i dont have a strong preference but if I can choose I choose that one)
* couple of rephrasings
* suggested HW of starting the 5 exercises of FCC (should take <10 mins and also will help to know how hard that was)

### CODE REVIEW

File: lessons/1_setup/index.html [hunk 1]
```diff
@@ -1,154 +1,212 @@
...
+                >https://code.visualstudio.com/download</a
+              ><small>* ask us about the recommended plugins</small>
+            </li>
+            <li>
+              Github: <a href="https://github.com/">https://github.com/</a>
+            </li>
+            <li>
+              freeCodeCamp:
+              <a href="https://www.freecodecamp.org/"
+                >https://www.freecodecamp.org/</a
```
Comment (MrTim) [inline lessons/1_setup/index.html:169]: Just curious: is it Prettier who format it's like that?)

**Review by MrTim — APPROVED** [review]

File: lessons/1_setup/index.html (hunk 1 above)
Comment (keponk) [reply lessons/1_setup/index.html:169]: haha yeah. do you use a diff one?

Comment (MrTim) [reply lessons/1_setup/index.html:169]: this is why I'm not using it, because after that the code looks, let's say, strange :)

Comment (keponk) [reply lessons/1_setup/index.html:169]: haha I understand, nevertheless: https://prettier.io/docs/en/why-prettier.html 

when it comes to styling I'm more of an adopter than a trendsetter :)

Comment (MrTim) [reply lessons/1_setup/index.html:169]: Maybe I should give it another try ;)

**Pull request merged**